lib/peep.rb

```diff
@@ -0,0 +1,53 @@
+require 'pg'
+require_relative './database_connection.rb'
+
+class Peep
+
+  attr_reader :id, :text, :time, :date, :user_id
+
+  def self.create(text:, user_id:)
+    Peep.set_environment
+
+    @connection = DatabaseConnection.connection
+    clock = Time.new
+    time = "#{clock.hour}:#{clock.min}:#{clock.sec}"
+    date = "#{clock.year}-#{clock.month}-#{clock.day}"
+
+    result = DatabaseConnection.query("INSERT INTO peeps (text, time, date, user_id) VALUES('#{text}', '#{time}', '#{date}', '#{user_id}')RETURNING id, text, time, date, user_id;")
```

Comment:
Inputting variables like this "#{text}', '#{time}', '#{date}', '#{user_id}" leaves the database open to SQL injection vulnerability (where users could inject their own queries into the database etc). You can use variables such as $1, $2, with an array returned after the query e.g      ;", [classhere.username, classhere.email])